### ISSUE TITLE
ExpandableSection - toggle text should be more descriptive

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7281]
**Describe the enhancement or change**
To follow the verbiage from design guidelines ("Informs the user what will be shown when the component is expanded") and to also recommend doing so when there may be multiple toggles on a page, the toggle text should be more descriptive and unique in examples. So instead of "Show more", "Show more basic example content" for the basic example and so on.

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye could you assign this to me. Thank You
